packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -270,16 +271,36 @@ class SkiaGoldClient {
         for(Map<String, dynamic> ignore in ignores) {
           final List<String> ignoredQueries = ignore['query'].split('&');
           final String ignoredPullRequest = ignore['note'].split('/').last;
-          if (ignoredQueries.contains('name=$testName') &&
-            ignoredPullRequest == pullRequest) {
-            ignoreIsActive = true;
-            break;
+          final DateTime expiration = DateTime.parse(ignore['expires']);
+          // The currently failing test is in the process of modification.
+          if (ignoredQueries.contains('name=$testName')) {
+            if (expiration.isAfter(DateTime.now())) {
+              ignoreIsActive = true;
+              break;
+            } else {
+              final StringBuffer buf = StringBuffer()
+                ..writeln('This test has an expired ignore in place, and the')
+                ..writeln('change has not been triaged.')
+                ..writeln('The associated pull request is:')
+                ..writeln('https://github.com/flutter/flutter/pull/$ignoredPullRequest');
+              throw NonZeroExitCode(1, buf.toString());
```

Comment:
This way provides insight into why it fails in this case (ignore has expired). It could fail based purely on the fact that there is not an ignore in place. i.e. My PR is making a golden change I did not expect!